### ISSUE TITLE
Failed to generate program "xxxx" using the Post Processor "Fanuc_RJ3_Dripfeed"

### ISSUE THREAD

**Issue by MarcoWithag** [inholland-compolab/FanucAutomatedManufacturing#12]
U
![image](https://user-images.githubusercontent.com/76169842/133986340-527cc6e3-8c6e-4acd-b6aa-0785b09bc092.png)

**Comment by BrinkCompositeEngineering** [comment]
Can you check if the Prog1.ls is actually made and saved in the location mentioned above? Looks like it can actually create the program but stops during making a connection with the robot for the dripfeed.

**Comment by MarcoWithag** [comment]
yes it does generate the LS and TP file, but when starting the dripfeed it gives an error.

**Comment by BrinkCompositeEngineering** [comment]
Can RoboDK connect to the robot? Is there a connection available?

**Comment by MarcoWithag** [comment]
yes the robot is connected to Robodk. I have just renewed the maintenance license. hope that the error will be fixed after installing the latest version of RoboDK.
